action.yml

```diff
@@ -10,7 +10,7 @@ inputs:
     required: false
     default: 'default'
   bundler:
-    description: 'The version of Bundler to install. Either none, 1, 2, latest or Gemfile.lock. The default tries Gemfile.lock and otherwise uses latest.'
+    description: 'The version of Bundler to install. Either none, 1, 2, latest or Gemfile.lock. In case of none and Gemfile.lock, the version is determined based on the BUNDLE_GEMFILE environment variable. If the variable is not set, it will default to Gemfile.lock. If a bundler option is not provided, it falls back to using "latest".'
```

Comment:
`none` actually does not install Bundler at all, and does not run any `bundle` command.

This feels too long and very hard to read due to line length, how about:
```yaml
    description: 'The version of Bundler to install. Either none, 1, 2, latest or Gemfile.lock. For Gemfile.lock, the version is determined based on the BUNDLED WITH section from the file ($BUNDLE_GEMFILE || Gemfile).lock. Defaults to Gemfile.lock if it exists and latest otherwise.
```

I think this would better discussed in the README though, where we can explain more intuitively what happens, and have more space: https://github.com/ruby/setup-ruby#bundler
Then this line does not really need to change.

Reply:
Thanks, @eregon for the review. I applied your suggestion and added a new paragraph to the README in https://github.com/ruby/setup-ruby/pull/74/commits/c23a60ff7419d5c7436ab69feaa08dfa5a7da235.